tests/unit/vcast-server.test.ts

```diff
@@ -60,7 +65,7 @@ describe("test server functions", () => {
     const result = await closeConnection("test/path");
     expect(result).toBe(true);
     expect(fetch).toHaveBeenCalledWith(
-      'http://localhost:60461/runcommand?request={"command":"closeConnection","path":"test/path"}'
+      'http://127.0.0.1:0/runcommand?request={"command":"closeConnection","path":"test/path"}'
```

Comment:
Could we define the host-name somewhere globally?

Reply:
I would suggest to transfer this request to 

https://github.com/vectorgrp/vector-vscode-vcast/pull/218

Because I needed to change the unit tests and the server calls there either way and this branch should be merged into the branch from #218